sqlmesh/core/engine_adapter/mixins.py

```diff
@@ -89,7 +89,7 @@ def _insert_overwrite_by_condition(
         for source_query in source_queries:
             with source_query as query:
                 query = self._order_projections_and_filter(query, columns_to_types, where=where)
-                columns = [exp.to_column(col) for col in columns_to_types]
+                columns = [exp.column(exp.to_identifier(col)) for col in columns_to_types]
```

Comment:
why do you need the to_identifier?

Reply:
we don't, I forgot to switch this thanks